examples/with_winit/src/test_scene.rs

```diff
@@ -75,13 +73,24 @@ pub fn render_svg(sb: &mut SceneBuilder, svg: &PicoSvg) {
     }
 }
 
-#[allow(unused)]
-pub fn render_tiger(sb: &mut SceneBuilder) {
-    use super::pico_svg::*;
-    let xml_str =
-        std::str::from_utf8(include_bytes!("../../assets/Ghostscript_Tiger.svg")).unwrap();
-    let svg = PicoSvg::load(xml_str, 6.0).unwrap();
-    render_svg(sb, &svg);
+pub fn render_svg_scene(
+    sb: &mut SceneBuilder,
+    scene: &mut Option<SceneFragment>,
+    xform: Affine,
+    svg: &str,
+) {
+    if scene.is_none() {
+        use super::pico_svg::*;
+        let start = Instant::now();
+        eprintln!("Starting to parse svg");
+        let svg = PicoSvg::load(svg, 6.0).unwrap();
```

Comment:
This could be a command line argument (it should be -1.5 for paris), but a more principled approach is to parse transforms in `<g>` elements.

Reply:
I made it a command line flag for now. Properly parsing svg is quite a task.

Some of the work in https://github.com/google/forma/blob/main/demo/src/demos/svg.rs might be useful to us, although `forma` is Apache only - no MIT option; so we need to be careful that we use it correctly if we do.